x/gov/module.go

```diff
@@ -128,6 +136,81 @@ func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, ak types.AccountKeeper,
 	}
 }
 
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(provideModuleBasic, provideModule),
+		appmodule.Invoke(invokeAddRoutes, invokeSetHooks))
+}
+
+func provideModuleBasic() runtime.AppModuleBasicWrapper {
+	return runtime.WrapAppModuleBasic(AppModuleBasic{})
+}
+
+func provideModule(
```

Comment:
For consistency I'd think we should use a struct named `govInputs` and `govOutputs`.